### ISSUE TITLE
android uiautomator enableMultiWindows bug?

### ISSUE THREAD

**Issue by TesterWa** [appium/appium-uiautomator2-server#500]
set enable MultiWindows：true，getPagesource Only return root，test application Wechat!

**Comment by TesterWa** [comment]
Yes ! Native element get roots It should be normal, but is there any problem with appium-ui2-server in generating the page source? Although the page source generation failed, you can get elements with Android native uiselector, provided that you call driver.source before calling findelement to empty windows in ui2-server. The cache. In order to return the element normally.

**Comment by TesterWa** [comment]
> 是原生元素吗？ 来自 as的`enableMultiWindows`返回视图
> 
> https://github.com/appium/appium-uiautomator2-server/blob/0111c096ac077772eef54b670e990cef42544580/app/src/main/java/io/appium/uiautomator2/utils/AXWindowHelpers.java#L86-L98
> 
> , 所以它只返回 root，那么 Android 的 AccessibilityNodeInfo 也可以显示它们。

Yes ! Native element get roots It should be normal, but is there any problem with appium-ui2-server in generating the page source? Although the page source generation failed, you can get elements with Android native uiselector, provided that you call driver.source before calling findelement to empty windows in ui2-server. The cache. In order to return the element normally.

**Comment by KazuCocoa** [comment]
Generally the result by Appium depends on https://developer.android.com/reference/android/app/UiAutomation#getWindows() and https://developer.android.com/reference/android/view/accessibility/AccessibilityNodeInfo. A potential reason was they did not retuirn entire page source.

[allowInvisibleElements](https://github.com/appium/appium-uiautomator2-driver/#settings-api) and [ignoreUnimportantViews](https://github.com/appium/appium-uiautomator2-driver/#settings-api) also could help https://github.com/appium/appium-uiautomator2-driver/#elements-cannot-be-found
